terraform/cloudwatch.tf.json.template.py

```diff
@@ -63,6 +63,13 @@ def prod_qualified_resource_name(name: str) -> str:
                                    ['aws_elasticsearch_domain.index'])
                 }
             }
+        },
+        {
+            'aws_sns_topic': {
+                'monitoring': {
```

Comment:
The references need to updated to include `data.`. Run `make plan` with `dev` selected to validate your changes.